src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC } from 'react';
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Typography,
+} from '@material-ui/core';
+import { TodoProps } from './typings';
+
+const Todo: FC<TodoProps> = ({id, todo, onRemove, onToggle}: TodoProps) => {
+  const { title, done } = todo;
+  const statusLabel = done ? 'Reativar' : 'Completar';
+  const opacity = done ? '0.5' : 1;
+
+  return (
+    <Card      
+      data-testid={`todo-${id}`}
+      key={id}
```

Comment:
Is key really necessary? I think is only necessary inside a `map` iteration.

Reply:
I copied and paste this card from `App.tsx` to the component. My mistake.
Resolved on ac40b7bd5b1418375c16876812c614ac7cb7f273